machine-learning/tutorials/MovieRecommendation/Program.cs

```diff
@@ -83,12 +83,15 @@ public static ITransformer BuildAndTrainModel(MLContext mlcontext, IDataView tra
             };
 
             est = est.Append(mlcontext.Recommendation().Trainers.MatrixFactorization(options));
+            // </SnippetAddAlgorithm>
 
+            // Train model
+            // <SnippetFitModel>
             Console.WriteLine("=============== Training the model ===============");
             ITransformer model = est.Fit(trainingDataView);
 
             return model;
-            // </SnippetTrainModel>
+            // </SnippetFitModel>
         }
 
         // Evaluate model
```

Comment:
**EvaluateModel:** 
mlcontext -> mlContext.

 EvaluateModel should be broken into the following snippets (remove spaces from snippet tags- there to display):
 // < SnippetTransformData >
 var prediction = model.Transform(testDataView);
 // < /SnippetTransformData >

// < SnippetEvaluate >
var metrics = mlcontext.Regression.Evaluate(prediction, label: DefaultColumnNames.Label, score: DefaultColumnNames.Score);
// < /SnippetEvaluate >

 //< SnippetDisplayMetrics >
Console.WriteLine("Rms: " + metrics.Rms.ToString());
Console.WriteLine("RSquared: " + metrics.RSquared.ToString());
 //< /SnippetDisplayMetrics >

**UseModelForSinglePrediction**
mlcontext -> mlContext.
predictionengine -> predictionEngine